Discussion Conclusion Reviewer Request (#84)
greenelab/annorxiver_manuscript: content/05.conclusion.md

## Discussion and Conclusions

BioRxiv is a constantly growing repository that contains life science preprints.

> Over 77% of bioRxiv preprints that were published were successfully detected within Pubmed Central's Open Access Corpus (PMCOA).

Reviewer: ```suggestion
Over 77% of bioRxiv preprints with a corresponding publication in our snapshot were successfully detected within Pubmed Central's Open Access Corpus (PMCOA).
```

> Over 77% of bioRxiv preprints that were published ended up within Pubmed Central's Open Access Corpus (PMCOA) which is a significant improvement compared to the earlier years [@doi:10.7287/peerj.preprints.3520v1].

Reviewer: ```suggestion
Over 77% of bioRxiv preprints that were published ended up within Pubmed Central's Open Access Corpus (PMCOA) [@doi:10.7287/peerj.preprints.3520v1].
```
Is this necessarily true? I thought we knew they were published, but some might be linked to sources _not_ in pubmed but that are captured by bioRxiv and represented in the crossref database?

> This highlights a beneficial shift towards more preprint authors publishing their work in an open access format.

Reviewer: ```suggestion
This suggests that most work from groups participating in the preprint ecosystem is now available in final form for literature mining and other applications.
```

> The majority of research involving bioRxiv focuses on the metadata of preprints; however, the language contained within these preprints has yet to be systematically examined.

Reviewer: ```suggestion
Most research on bioRxiv preprints has examined their metadata; we examine the text content as well.
```

Throughout this work, we sought to analyze the language within these preprints and understand how it changes in response to peer review.

Our global corpora analysis found that writing within bioRxiv is consistent with the biomedical literature in the PMCOA repository, suggesting that bioRxiv is linguistically similar to PMCOA.
Token-level analyses between bioRxiv and PMCOA suggested that research fields drive significant differences; e.g., more patient-related research is prevalent in PMCOA than bioRxiv.
This observation is expected as preprints focused on medicine are supported by the complementary medRxiv repository [@url:https://www.medrxiv.org/].
Token-level analyses for preprints and their corresponding published version suggest that peer review may focus on data availability and incorporating extra sections for published papers; however, future studies are needed to ascertain individual token level changes as preprints venture through the publication process.

> Furthermore, it would be interesting to perform a token comparison between  preprints and accepted author manuscripts within Pubmed Central as this may highlight non-journal related differences. 

Reviewer: ```suggestion
One future avenue of research could examine the differences between only preprints and accepted author manuscripts within Pubmed Central to identify changes prior to journal publication. 
```

Document embeddings are a versatile way to examine language contained within preprints, understanding peer review's effect on preprints, and provide extra functionality for preprint repositories.

> We used a linear approach to generate document embeddings as our main goal was to increase model interpretability; however, future work could look into incorporating more advanced models to generate document vectors such as Doc2Vec[@arxiv:arXiv:1405.4053] or BERT [@doi:10.1093/bioinformatics/btz682].

Reviewer: ```suggestion
Our approach to generate document embeddings was focused on interpretability instead of predictive performance; however, using more advanced strategies to generate document vectors such as Doc2Vec[@arxiv:arXiv:1405.4053] or BERT [@doi:10.1093/bioinformatics/btz682] should increase predictive performance.
```

Examining linguistic variance within document embeddings of life science preprints revealed that the largest source of variability was informatics.
This observation bisects the majority of life science research categories that have integrated preprints within their publication workflow.
This embedding space could also be used to quantify sentiment trends or other linguistic features.

> Furthermore, future applications such as uncovering latent scientific knowledge similar to [@doi:10.1038/s41586-019-1335-8] could be applied to this embedding space.

Reviewer: ```suggestion
Furthermore, methodologies for uncovering latent scientific knowledge [@doi:10.1038/s41586-019-1335-8] may be applicable in this embedding space.
```

> Along with anaylyzing lingusitic variation, future work could utilize this embedding space to quantify sentiment trends such as authors writing preprints with a more positive language over time.

Reviewer: ```suggestion
This embedding space could also be used to quantify sentiment trends or other linguistic features.
```

> Furthermore, future applications similar to [@doi:10.1038/s41586-019-1335-8] could be applied to this embedding space.

Reviewer: What is the application? Someone shouldn't have to read the entire reference to understand the sentence :)
